src/setup/txt_joybinput.c

```diff
@@ -60,6 +60,37 @@ static int *all_joystick_buttons[NUM_VIRTUAL_BUTTONS] =
     &joybautomap,
 };
 
+// For indirection so that we're not dependent on item ordering in the
+// SDL_GameControllerButton enum.
+static const int gamepad_buttons[GAMEPAD_BUTTON_MAX] =
```

Comment:
`GamepadButton_t` has to inherit from `SDL_GameControllerButton` since I'm using it to capture binds. It does not depend on the `SDL_GameControllerButton` enum ordering, just the values.